README.md

```diff
@@ -28,12 +28,28 @@ Shiori is a simple bookmarks manager written in Go language. Intended as a simpl
 
 ## Installation
 
+### Linux
+
 You can download the latest version of `shiori` from [the release page](https://github.com/RadhiFadlillah/shiori/releases/latest), then put it in your `PATH`. If you want to build from source, make sure `go` is installed, then run :
 
 ```
 go get github.com/RadhiFadlillah/shiori
 ```
 
+### Windows
+
+In order to build this project, you will need to have access to a Linux machine or a Windows Subsystem for Linux environment. The following commands work under Ubuntu 16.04 LTS:
```

Comment:
WSL is necessarily needed to install mingw for gcc. see https://github.com/mattn/go-sqlite3/issues/212#issuecomment-273531789

Reply:
I didn't mention that it was, but rather specifically mentioned that it does work there in case that is the preferred method for compiling it